roles/influxdb/tasks/main.yml

```
@@ -16,3 +16,6 @@
   service:
     name: influxdb
     state: started
+
+- include_tasks: bucket.yml
```

```suggestion
- include_tasks: bucket.yml
```
Remove the blank line in the end. YAML linter will always show that as lint.